src/main/resources/TotemCode/TotemMacros.xml

```diff
@@ -311,5 +311,27 @@
 &lt;input type="hidden" id="redirectURL" value="$doc.getURL('view','showEdit=true')"/&gt;
 &lt;input type="hidden" id="showIframe" value="1"/&gt;
 #end
+
+#**
+* Display the totem actions links : Edit totem, documentation ...
+*#
+#macro(displayTotemActionsLinks)
+#set($totemViewURL = $doc.getURL())
+#set($totemEditURL = $doc.getURL('view','showEdit=true'))
+#set($totemDocURL = $xwiki.getURL('TotemCode.Documentation','view'))
+{{html clean="false" wiki="true"}}
```

Comment:
I see. Then you should put an empty line before and after the call of `#displayTotemActionsLinks()`. The problem is that the HTML macro is used inline meanwhile it should be seen as a separate "block".